data_steward/tools/create_rdr_snapshot.py

```diff
@@ -179,5 +197,64 @@ def create_datasets(client, rdr_dataset, release_tag):
     return {'clean': rdr_clean, 'staging': rdr_staging, 'sandbox': rdr_sandbox}
 
 
+def mapping(client, input_dataset_id, domain_table):
+    """
+    Create and load a  mappping table that stores the domain id and src_id.
+    Note: Overwrites destination table if it already exists
+
+    :param client: a BigQueryClient
+    :param input_dataset_id: identifies dataset containing the domain table.
+    :param domain_table: name of domain table to create mapping table for.
+    :return:
+    """
+    query = mapping_query(domain_table, input_dataset_id, client.project)
+    mapping_table = mapping_table_for(domain_table)
+    logging.info(f'Query for {mapping_table} is {query}')
+    query_job = client.query(query)
+    query_job.result()
+
+
+def mapping_query(table_name, dataset_id=None, project_id=None):
```

Comment:
We do not have to create mapping tables for all the tables that RDR gives us, we should only create _mpping_tables for only the tables that we are creating earlier. we have MAPPED_CLINICAL_DATA_TABLES + PERSON + DEATH. for all the other tables like fact_relationship and metadata, we can ignore creating mapping tables.

Reply:
I believe we are not creating a mapping table for DEATH table.